global/get-modifed-folders/action.yml

```diff
@@ -0,0 +1,197 @@
+name: "📁 Get modified directory and sparse checkout"
+description: "Retrieve all modified directory and perform a sparse checkout."
+
+
+inputs:
+  starting_index:
+    description: '(Optional) Starting index for path filtering'
+    default: '/'
+    required: true
```

Comment:
is correct to say Optional, but put required?

Reply:
yes it's ok to be optional the starting index, because most people will check starting by root dir